### DISCUSSION TITLE
What does pod annotation `io.kubernetes.cri-o.TTY/x: "false"` do?

### DISCUSSION THREAD

**Discussion by dilyanpalauzov** [containers/podman#15228] [General]
```console
$ podman generate kube …
apiVersion: v1
kind: Pod
metadata:
  annotations:
    io.kubernetes.cri-o.TTY/vigorousshannon: "false"
…
```
Please describe what does the annotation `io.kubernetes.cri-o.TTY/x: true`.  I find no documentation for it on cri-o.

```console
$ podman version
Client:       Podman Engine
Version:      4.2.0-dev
API Version:  4.2.0-dev
Go Version:   go1.18.4
Git Commit:   ccbb8e7444423306691f8ec441ed09387718d6e5
Built:        Thu Aug  4 14:05:34 2022
OS/Arch:      linux/amd64
```

**Comment by rhatdan** [comment]
This looks like something you added to your pod or container and then podman added it to the kubernetes yaml.

Moving this to a discussion.

**Comment by dilyanpalauzov** [comment]
`"io.kubernetes.cri-o.TTY": "false"` appears also in [man podman-container-inspect](https://docs.podman.io/en/latest/markdown/podman-container-inspect.1.html#example).

**Comment by rhatdan** [comment]
Your correct we do add this, and I have no idea why.
@baude @mheon Do you know why this annotation is added?

It looks like it was added by Brent?

**Comment by mheon** [comment]
Indicates if the container has a TTY. A holdover from the CRI-O days,
largely; we kept it in Libpod since we planned to migrate CRI-O to it
eventually. Largely vestigial at this point; might be used by K8S, but not
by us.
